### PR TITLE
Updates to root project readme description

### PR DESCRIPTION
**Pull request by shanhix1-personal**

### CODE REVIEW

**Comment by shanhix1-personal** [comment]
@wejama @sethyboy20 @Em-Anderson please review & leave comments in this PR to be addressed

**Comment by wesahearn** [comment]
App names:

ReciPal
ChronoMeal
Crumbee
Foodle
MacroMeal
Tangelo
BitBurger
ReciCal
JumblyMeal
ReciPlan
MyFoodie
Honeycrisp
FoodFetch
TallyMeal
Organic
SimpliMeal
FlashFood
Mallow
Mel
Foodrabbit
Calxifer
Plumbago

**Comment by wejama** [comment]
App names:

ReciPal
ChronoMeal
Crumbee
Foodle
MacroMeal
Tangelo
BitBurger
ReciCal
JumblyMeal
ReciPlan
MyFoodie
Honeycrisp
FoodFetch
TallyMeal
Organic
SimpliMeal
FlashFood
Mallow
Mel
Foodrabbit
Calxifer
Plumbago

File: README.md [hunk 1]
```diff
@@ -2,15 +2,30 @@
 by the SWErve team
 
 ## Project Name
-[insert project name here]
+Name ideas below:
+* Recipe Planner
+* Meal Dreamer
```
Comment (Em-Anderson) [inline README.md:7]: I like this name the most!

**Review by Em-Anderson — APPROVED** [review]

**Review by sethyboy20 — COMMENTED** [review]
looks good so far. for name, I would take Wes's suggestions into account.  "ReciPal", "Crumbee", "Mallow" would be my top 3

**Review by sethyboy20 — COMMENTED** [review]
Fix Wes's username to "wejama"

**Pull request merged**